db/sql/045_create_vat_rates.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS vat_rates (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR(32),
+	rate NUMERIC,
+
+	CHECK (length(trim(name)) > 0),
+	CHECK (rate >= 0)
+);
+
+INSERT INTO
```

Comment:
This adds rows every time the app is restarted. See https://github.com/alphagov/paas-billing/pull/21#issuecomment-372408162